docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/StochasticDualCoordinateAscent.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.MulticlassClassification
+{
+    public static class StochasticDualCoordinateAscent
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create in-memory examples as C# native class.
+            var examples = DatasetUtils.GenerateRandomMulticlassClassificationExamples(1000);
```

Comment:
Unfortunately, `GenerateRandomMulticlassClassificationExamples` is not searchable on the doc site, so the only way to fully learn this pipeline is to clone ML.NET.

Reply:
the type is visible if they use the example, and they can inspect the values with the debugger, BUT moving Featurization into the Samples Utils is a real problem.. 

---
In reply to: [261013871](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261013871) [](ancestors = 261013871)

Reply:
No, we can't expect user will have visual studio, on for example, Linux. I'd say the best case is that user knows everything they need after `reading` this example. In addition, those functions are not searchable on ML.NET doc site, which means a big hole to users.

Reply:
the function and type are both searchable:
* [https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.samplesutils.datasetutils.generaterandommulticlassclassificationexamples?view=ml-dotnet#Microsoft_ML_SamplesUtils_DatasetUtils_GenerateRandomMulticlassClassificationExamples_System_Int32](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.samplesutils.datasetutils.generaterandommulticlassclassificationexamples?view=ml-dotnet#Microsoft_ML_SamplesUtils_DatasetUtils_GenerateRandomMulticlassClassificationExamples_System_Int32)_
* [https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.samplesutils.datasetutils.multiclassclassificationexample?view=ml-dotnet](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.samplesutils.datasetutils.multiclassclassificationexample?view=ml-dotnet)

There no featurization here, b/c we're using randomly generated data. The featurization issue happens for loading data from file.

---
In reply to: [261051041](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261051041) [](ancestors = 261051041,261013871)

Reply:
Please see my latest comment on #2627. Removing SamplesUtils is a design decision that needs to be made first. 

Please see also see my response in another comment where I have the doc links. All of DatasetUtils are searchable:
[https://docs.microsoft.com/en-us/dotnet/api/?view=ml-dotnet&term=Microsoft.ML.SamplesUtils.DatasetUtils](https://docs.microsoft.com/en-us/dotnet/api/?view=ml-dotnet&term=Microsoft.ML.SamplesUtils.DatasetUtils)


---
In reply to: [261068842](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261068842) [](ancestors = 261068842)

Reply:
My bad. Even if it's searchable, it still has no meaningful document at [this page](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.samplesutils.datasetutils.generaterandommulticlassclassificationexamples?view=ml-dotnet).

Reply:
Our documentation coverage is low but we're actively working on it, hence this PR. So that page will become meaningful eventually.

---
In reply to: [261718177](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261718177) [](ancestors = 261718177)

Reply:
Please do not put things separated if they are considered a whole example. The organization of the entire documentation will never be organized and learned in a structured way so that user can learn them step-by-step --- this is how computer science world works.

Reply:
As pointed by Shaauheen, let's keep the samples as is for V1. Post V1, we can address this by proper discussions that were canceled in favor of API work. For now, some sample for V1, is better than no sample. 

---
In reply to: [261763964](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261763964) [](ancestors = 261763964)